genai-perf/genai_perf/config/input/config_endpoint.py

```diff
@@ -39,8 +42,8 @@ def __init__(self) -> None:
         )
         self.backend: Any = ConfigField(
             default=EndPointDefaults.BACKEND,
-            choices=OutputFormat,
-            verbose_template_comment="When using the \"triton\" service-kind, this is the backend of the model.\
+            choices=list(OutputFormat)[:2],
```

Comment:
Why are the other output formats not allowed? I see this is a bug I introduced when creating the config (comparing to what is in parser). I'm wondering if it would be cleaner to have a different enum that specifies the valid user inputs for backends instead?

Reply:
The other output formats are not Triton backends. Since the backend names == output format, I think that's the design that was used. If you look at the enum, you can see that it's broken up into categories with comments.

I have looked into redesigning this enum before, but it was never enough of a priority. If it's important to you, we can create a ticket to look into refactoring the enum.

Reply:
Probably not worth the effort.

Reply:
It will be a source of a bug at _some_ point. 
I think we make the ticket.

Reply:
Thank you both for calling this out! Created TPA-1070.